docs/source/tutorial/train_loop.rst

```diff
@@ -0,0 +1,328 @@
+How to write a training loop in Chainer
+```````````````````````````````````````
+
+.. currentmodule:: chainer
+
+In this tutorial section, we will learn how to train a deep neural network to classify images of hand-written digits in the popular MNIST dataset. This dataset contains 50,000 training examples and 10,000 test examples. Each example is a set of a 28 x 28 greyscale image and a corresponding class label. Since the digits from 0 to 9 are used, there are 10 classes for the labels.
+
+Chainer provides a feature called :class:`~chainer.training.Trainer` that can simplify the training procedure of your model. However, it is also good to know how the training works in Chainer before starting to use the useful :class:`~chainer.training.Trainer` class that hides the actual processes. Writing your own training loop can be useful for learning how :class:`~chainer.training.Trainer` works or for implementing features not included in the standard trainer.
+
+The complete training procedure consists of the following steps:
+
+1. Prepare a dataset.
+2. Create an iterator for the dataset.
+3. Write a training loop that performs the following operations in each iteration:
+    1. Retrieve a set of examples (mini-batch) from the training dataset.
```

Comment:
- Can you change the numbering of the sub items (e.g. i, ii, ... or a, b, ... )?
- The line "*3. Write a ...*" is rendered in bold font, which should be fixed.